rotkehlchen/chain/ethereum/manager.py

```diff
@@ -1195,3 +1195,23 @@ def get_basic_contract_info(self, address: ChecksumEthAddress) -> Dict[str, Any]
 
         self.contract_info_cache[address] = info
         return info
+
+    def _get_basic_contract_info_fallback(
+            self,
+            output: List[Tuple[bool, bytes]],
+            properties: Tuple[str, str, str],
+            contract: EthereumContract,
+    ) -> List:
+        """Acts a fallback for `get_basic_contract_info` when a RemoteError happens.
+        'decimals' property defaults to 18.
+        """
+        decoded_contract_info = []
+        for x, method_name in zip(output, properties):
```

Comment:
It shouldn't happen since we are the ones calling this function but maybe add an assert here that output and properties need to have the same length? What do you think?

Reply:
It's always three though.
https://github.com/rotki/rotki/blob/develop/rotkehlchen/chain/ethereum/manager.py#L1171

Reply:
yep I know. Maybe the type is just enough and the linter wil tell us